extensions/blocks/calendly/index.js

```diff
@@ -41,4 +43,17 @@ export const settings = {
 			url: 'https://calendly.com/wordpresscom/jetpack-block-example',
 		},
 	},
+	transforms: {
+		from: [
+			{
+				type: 'raw',
+				isMatch: node => node.nodeName === 'P' && REGEX.test( node.textContent ),
```

Comment:
As we're testing `textContent` what would happen if someone inserted a link to a Calendly URL in the middle of a paragraph?

Reply:
I think it only works on a paste, and only when its the initial block (raw) - once you start typing the block becomes a paragraph. Without this change Gutenberg tries to treat it like an embed...

Reply:
Gotcha! I was thinking that someone might paste a URL in and the pragraph would change.